lib/tasks/sample_data.rake

```diff
@@ -0,0 +1,656 @@
+# The sample data generated by this task is supposed to save some time during
+# manual testing. It is not meant to be complete, but we try to improve it
+# over time. How much is hardcoded here is a trade off between developer time
+# and tester time. We also can't include secrets like payment gateway
+# configurations in the code since it's public. We have been discussing this for
+# a while:
+#
+# - https://community.openfoodnetwork.org/t/seed-data-development-provisioning-deployment/910
+# - https://github.com/openfoodfoundation/openfoodnetwork/issues/2072
+#
+namespace :openfoodnetwork do
+  namespace :sample_data do
+    desc 'load sample data for development or staging'
+    task load: :environment do
+      raise "Please run `rake db:seed` first." unless seeded?
+
+      users = UserFactory.new.create_samples
+
+      enterprises = EnterpriseFactory.new.create_samples(users)
+
+      PermissionFactory.new.create_samples(enterprises)
+
+      FeeFactory.new.create_samples(enterprises)
+
+      ShippingMethodFactory.new.create_samples(enterprises)
+
+      PaymentMethodFactory.new.create_samples(enterprises)
+
+      TaxonFactory.new.create_samples
+
+      products = ProductFactory.new.create_samples(enterprises)
+
+      InventoryFactory.new.create_samples(products)
+
+      OrderCycleFactory.new.create_samples
+
+      CustomerFactory.new.create_samples(users)
+
+      GroupFactory.new.create_samples
+    end
+
+    def seeded?
+      Spree::User.count > 0 &&
+        Spree::Country.count > 0 &&
+        Spree::State.count > 0
+    end
+
+    module Logging
+      private
+
+      def log(message)
+        puts "[openfoodnetwork:sample_data:load] #{message}"
+      end
+    end
```

Comment:
we could use the rails logger as any part of Rails. [TaggedLogging](https://apidock.com/rails/v3.2.13/ActiveSupport/TaggedLogging) is a perfect fit for this.

Reply:
Ah, yes, that looks quite apt. But I don't see the benefit in this simple scenario. The change would be this:
```diff
-  puts "[openfoodnetwork:sample_data:load] #{message}"
+  @logger ||= ActiveSupport::TaggedLogging.new(Logger.new(STDOUT))
+  @logger.tagged("openfoodnetwork:sample_data:load") { @logger.info(message) }
```
That's much more complicated. What is the added value?

Reply:
It's more about using a logger rather than puts. We have logging built-in in Rails that we could leverage. You can then add those tags using the formatter attribute of the Rails logger for instance.

https://logmatic.io/blog/rails-logger-best-practices/ seems to be a good post about the topic.

Reply:
I changed it now as you suggested, but I don't see any added value in this case.